app/assets/javascripts/spree/backend/webhooks_subscribers.js

```diff
@@ -0,0 +1,13 @@
+$(document).ready(function () {
```

Comment:
@aleksandarpetrushev please let's not use jquery code anymore, we're using stimulus for Js code in admin